rustler/src/codegen_runtime.rs

```diff
@@ -16,14 +16,14 @@ pub use crate::wrapper::{
 pub use rustler_sys::{TWinDynNifCallbacks, WIN_DYN_NIF_CALLBACKS};
 
 pub unsafe trait NifReturnable {
-    unsafe fn as_returned(self, env: Env) -> NifReturned;
```

Comment:
Clippy advises that `as_` should take `&self` or `&mut self` (see [here](https://rust-lang.github.io/rust-clippy/master/#wrong_self_convention)). As we want to take ownership here (or am I mistaken?), the function should be prefixed with `into_` instead.

Reply:
This sounds right to me 👍

`codegen_runtime` is `pub` but hidden, so I wouldn't see this as a breaking change.

Reply:
Agreed, `codegen_runtime` is public because it is used by our macros, but is not a documented or stable API outside of that.